tests/src/python/test_qgsvectorlayereditutils.py

```diff
@@ -266,6 +282,158 @@ def testAddRingV2SelectedFeatures(self):
+
+        self.assertTrue(vle.moveVertexV2(QgsPoint(3, 4, 5, 6), f1.id(), 0))
+        f1 = layer.getFeature(1)
+
+        self.assertEqual(f1.geometry().constGet().x(), 3)
+        self.assertEqual(f1.geometry().constGet().y(), 4)
+        self.assertEqual(f1.geometry().constGet().m(), 6)
+        self.assertFalse(f1.geometry().constGet().is3D())
+        self.assertTrue(f1.geometry().constGet().isMeasure())
+
+        # Add a non-M point and check that M get added on move
+        f2 = QgsFeature(layer.fields(), 2)
+        f2.setGeometry(QgsGeometry.fromWkt('Point(0 0)'))
+        self.assertTrue(pr.addFeatures([f2]))
+        self.assertEqual(layer.featureCount(), 2)
+
+        self.assertFalse(f2.geometry().constGet().isMeasure())
+
+        self.assertTrue(vle.moveVertexV2(QgsPoint(3, 4, 5, 6), f2.id(), 0))
+        f2 = layer.getFeature(2)
+
+        self.assertTrue(f2.geometry().constGet().isMeasure())
+        self.assertFalse(f2.geometry().constGet().is3D())
+        self.assertEqual(f2.geometry().constGet().x(), 3)
+        self.assertEqual(f2.geometry().constGet().y(), 4)
+        self.assertEqual(f2.geometry().constGet().m(), 6)
+
+    def testMoveVertexPointZM(self):
+        layer = QgsVectorLayer("PointZM", "pointZM", "memory")
+        self.assertTrue(layer.startEditing())
+        self.assertTrue(layer.isValid())
+
+        pr = layer.dataProvider()
+        f1 = QgsFeature(layer.fields(), 1)
+        f1.setGeometry(QgsGeometry.fromWkt('PointZM(0 0 0 0)'))
+        self.assertTrue(pr.addFeatures([f1]))
+        self.assertEqual(layer.featureCount(), 1)
+
+        vle = QgsVectorLayerEditUtils(layer)
+
+        self.assertTrue(vle.moveVertexV2(QgsPoint(3, 4, 5, 6), f1.id(), 0))
+        f1 = layer.getFeature(1)
+
+        self.assertEqual(f1.geometry().constGet().x(), 3)
+        self.assertEqual(f1.geometry().constGet().y(), 4)
+        self.assertEqual(f1.geometry().constGet().z(), 5)
+        self.assertEqual(f1.geometry().constGet().m(), 6)
+        self.assertTrue(f1.geometry().constGet().is3D())
+        self.assertTrue(f1.geometry().constGet().isMeasure())
+
+        # Add a non-ZM point and check that Z and M get added on move
+        f2 = QgsFeature(layer.fields(), 2)
+        f2.setGeometry(QgsGeometry.fromWkt('Point(0 0)'))
+        self.assertTrue(pr.addFeatures([f2]))
+        self.assertEqual(layer.featureCount(), 2)
+
+        self.assertFalse(f2.geometry().constGet().isMeasure())
+        self.assertFalse(f2.geometry().constGet().is3D())
+
+        self.assertTrue(vle.moveVertexV2(QgsPoint(3, 4, 5, 6), f2.id(), 0))
+        f2 = layer.getFeature(2)
+        self.assertTrue(f2.geometry().constGet().isMeasure())
+        self.assertTrue(f2.geometry().constGet().is3D())
+
+        self.assertEqual(f2.geometry().constGet().x(), 3)
+        self.assertEqual(f2.geometry().constGet().y(), 4)
+        self.assertEqual(f2.geometry().constGet().z(), 5)
+        self.assertEqual(f2.geometry().constGet().m(), 6)
+
```

Comment:
```suggestion
```

Reply:
@domi4484 As far as I understand you, this are tests added in this commit but concerning another implementation. So we just can remove it?